jormungandr/src/main.rs

```diff
@@ -337,11 +337,10 @@ fn bootstrap(initialized_node: InitializedNode) -> Result<BootstrappedNode, star
 
     let block0_explorer = block0.clone();
 
-    // TODO: we should get this value from the configuration
-    let block_cache_ttl: Duration = Duration::from_secs(5 * 24 * 3600);
+    let cache_capacity = 10_240;
```

Comment:
Should be way higher than this, probably x10

Reply:
```suggestion
    let cache_capacity = 102_400;
```